Header header
ethercat_hardware/BoardInfo board_info
ethercat_hardware/ActuatorInfo actuator_info
ethercat_hardware/MotorTraceSample[] samples